app/app_upgrades.go

```diff
@@ -69,7 +70,7 @@ func setHandlerForVersion_1_1(app *App) {
 
 	if upgradeInfo.Name == upgradeVersion && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
 		storeUpgrades := storetypes.StoreUpgrades{
-			Added: []string{authz.ModuleName, group.ModuleName, addressbookTypes.ModuleName, marketplaceTypes.ModuleName},
+			Added: []string{authz.ModuleName, group.ModuleName, AddressBookModuleName, MarketplaceModuleName},
```

Comment:
we're currently running node version 1.1  shouldn't these changes be applied to a `setHandlerForVersion_1_2` function instead for the new upgrade?

Reply:
We've removed the 3 modules from the codebase and thus making their `ModuleName` inaccessible, so these changes are solely intended to making the code compiles, and not related to the upgrade process.

Reply:
related to this message as well: https://cudocrypto.slack.com/archives/C05RC14GBEW/p1703155003933409?thread_ts=1703154904.665749&cid=C05RC14GBEW

But please let me know if what I'm saying doesn't make sense.

Reply:
@jgtormo 
> Shouldn't we leave the current version as it is, and start including this in a new version?

Yes, that is essentially what I am doing. My changes are similar to this of Osmosis:  https://github.com/osmosis-labs/osmosis/blob/ec3603396e022403dfa7e7765f819c5d304d3003/app/upgrades/v9/constants.go#L16-L26

> related to this message as well: https://cudocrypto.slack.com/archives/C05RC14GBEW/p1703155003933409?thread_ts=1703154904.665749&cid=C05RC14GBEW

I can't access the link, it requires a Cudos email.

Reply:
isn't [this line](https://github.com/osmosis-labs/osmosis/blob/ec3603396e022403dfa7e7765f819c5d304d3003/app/upgrades/v9/constants.go#L25) explicitly signalling that in the upgrade from v8 to v9 the `claim` module will be removed? Rather than modifying to v8 file and removing its reference from there

Reply:
@jgtormo I think there is a misunderstanding here, these changes are only to make the cleanup compatible with old code base.

So, when @tropicaldog removed those three modules, it created **syntax errors** since there was no longer those module names. He had to add those names in to fix syntax errors. It is not related to upgrades.

Or, perhaps, what you mean is to create a separate file for upgrade constants? Sorry if there are any misunderstandings

Reply:
yes, and I apologise as well if I'm not being clear enough or what I say just doesn't make sense. What I'm trying to say is that I was thinking the cleanup would be done as part of an upgrade, and that everything that references the currently running code with these modules in would stay unaffected. Mainnet is currently running v1.1 and [`setHandlerForVersion1_1`](https://github.com/CudoVentures/cudos-node/blob/12a356cca43f15e89d7b883302f3dba3b6e58fcd/app/app_upgrades.go#L43C23-L43C23) is, as I understand, specific to that version/upgrade to that version. So rather than modifying code that specifically relates to a release that all validators are currently running on mainnet, what I understood this file to be and what I seem to understand Osmosis do is signal in a new upgrade that some modules are being deleted.

In other words, shouldn't removing these three modules create syntax errors, and shouldn't that be the expected behaviour? Because in the v1.1 release they are there, and this function relates to v1.1 so if they are gone that should be an issue if trying to run that version.

Reply:
had a chat with @nbaum and he has some different comments, but closing this for now and merging.